newsroom/wire/search.py

```diff
@@ -25,6 +27,13 @@ def get_aggregations():
     return app.config.get("WIRE_AGGS", {})
 
 
+class TimeFilter(TypedDict):
+    name: str
+    default: bool
+    query: str
```

Comment:
might be good idea if the query is set as:
```py
class DateRangeQuery(TypedDict, total=False):
    gt: str
    gte: str
    lt: str
    lte: str
    time_zone: str
```
and then we could just apply whatever is set there